.github/workflows/push.yml

```diff
@@ -22,3 +22,6 @@ jobs:
   pack:
     needs: [test]
     uses: ./.github/workflows/pack.yml
+  publish:
+    needs: [pack]
+    uses: ./.github/workflows/pack.yml
```

Comment:
Why pack and publish has the same workflow

Reply:
to see it works with publish the secret